repositories.yml

```diff
@@ -3,49 +3,173 @@ section:
       - name: cyberark/conjur
         url: https://github.com/cyberark/conjur
         description: Conjur OSS server. Conjur comes built-in with custom authenticators
           for Kubernetes, OpenShift, AWS IAM, OIDC, and more.
         certification: "certified"
-        after: v1.3.5
+        version: v1.4.4
         upgrade_url: https://docs.cyberark.com/Product-Doc/OnlineHelp/AAM-DAP/Latest/en/Content/Deployment/Upgrade/upgrade-intro.htm
       - name: cyberark/conjur-oss-helm-chart
         url: https://github.com/cyberark/conjur-oss-helm-chart
         description: Helm chart for deploying Conjur OSS.
         version: v1.3.7
-      - name: cyberark/secretless-broker
-        url: https://github.com/cyberark/secretless-broker
-        description: Secretless Broker is a connection broker which relieves client applications of the need to directly handle secrets to target services such as databases, web services, SSH connections, or any other TCP-based service.
-        version: v1.4.2
+        certification: "trusted"
 #      - name: cyberark/conjur-aws
 #        url: https://github.com/cyberark/conjur-aws
 #        description: AWS CloudFormation templates for deploying Conjur OSS.
+#        certification: "community"
+#        version: v{no tags, can't include in release}
+        
   - name: Conjur SDK
     description: Conjur Command Line Interface (CLI) and Client Libraries
     repos:
       - name: cyberark/conjur-cli
         url: https://github.com/cyberark/conjur-cli
         description: Conjur Ruby CLI
         certification: "certified"
-        version: v6.2.1
-#      - name: cyberark/conjur-api-dotnet
-#        url: https://github.com/cyberark/conjur-api-dotnet
-#        description: Conjur .Net Client Library
+        version: v6.2.0
+      - name: cyberark/conjur-api-dotnet
+        url: https://github.com/cyberark/conjur-api-dotnet
+        description: Conjur .Net Client Library
+        certification: "community"
+        version: v1.1.1
       - name: cyberark/conjur-api-go
         url: https://github.com/cyberark/conjur-api-go
         description: Conjur Golang Client Library
+        certification: "certified"
+        version: v0.5.2
       - name: cyberark/conjur-api-java
         url: https://github.com/cyberark/conjur-api-java
         description: Conjur Java Client Library
+        certification: "community"
+        version: v1.1
       - name: cyberark/conjur-api-python3
         url: https://github.com/cyberark/conjur-api-python3
         description: Conjur Python Client Library
         certification: "community"
+        version: v0.0.5
       - name: cyberark/conjur-api-ruby
         url: https://github.com/cyberark/conjur-api-ruby
         description: Conjur Ruby Client Library
-        version: v4.10.2
+        version: v5.3.0
+        certification: "certified"
+        
+  # for fun and to help sanity check, we'll do these back two versions for now
+  # TODO: remove this comment later
```

Comment:
This comment block seems like it could use a better description. Why are we going back two version vs 1 ? vs 3? What are we sanity checking? When do we remove this comment? Also needs capitalization for beginning word.